content/src/fieldConfig/xray.json

```diff
@@ -39,6 +39,14 @@
     "serialNumber": "Serial Number: ${serialNumber}",
     "annualFee": "Annual Fee: ${annualFee}",
     "issuingAgencyText": "Issuing Agency",
-    "equipmentText": "Equipment"
+    "equipmentText": "Equipment",
+    "ariaLabel": "X-Ray Registration Task"
+  },
+  "xrayRenewal": {
+    "callToActionSecondaryText": "Check My Registration Status",
+    "callToActionPrimaryText": "Renew My X-Ray Machine Registration",
```

Comment:
pebble - this is being used in who places (xray-reneway.md file), do we need in both places?